templates/gulpfile.js.ts

```diff
@@ -3,8 +3,8 @@ import { TemplateFunction } from "../src/lib/createAdapter";
 export = (answers => {
 
 	const useTypeScript = answers.language === "TypeScript";
-	const useReact = answers.adminReact === "yes";
-	const skipWordsJs = useReact;
+	const hasTab = answers.adminFeatures && answers.adminFeatures.indexOf("tab") > -1;
+	const skipWordsJs = answers.adminReact === "yes" && (!hasTab || answers.tabReact === "yes");
```

Comment:
I'd use the same logic as in words.js here.

Reply:
It is, except for the missing isWidget test which wasn't there before. Not sure if we should add this.

Reply:
On second look I'm okay with this.